ludwig/schema/encoders/text_encoders.py

```diff
@@ -2574,7 +2574,7 @@ def module_name():
     )
 
     use_pretrained: bool = schema_utils.Boolean(
-        default=False,
+        default=True,
         description="Whether to use the pretrained weights for the model. If false, the model will train from "
         "scratch which is very computationally expensive.",
         parameter_metadata=ENCODER_METADATA["FlauBERT"]["use_pretrained"],
```

Comment:
We should always set use_pretrained to True by default right?

Reply:
Good catch.